### PR TITLE
Create digital-goods-are-different.md

### PR DESCRIPTION
**Pull request by theo-cox**
[m]: create digital-goods-are-different.md

### CODE REVIEW

File: content/notes/digital-goods-are-different.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Why digital goods are different
```
Comment (rufuspollock) [inline content/notes/digital-goods-are-different.md:1]: Probably wanted a

```
---
created: 2023-04-04
---
```

section at the top.

File: content/notes/digital-goods-are-different.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Incentives (at least according to orthodox economics) are a matter of cost-benefit analysis. At the most basic level rational agents will do something such as produce a good if there is a net benefit to doing so i.e. the benefits outweigh the costs (including opportunity costs[^3]). If the net benefits to wider society of producing a good outweigh the net benefit to private producers, we get market failure - the good will be underproduced versus the socially optimal level. 
+
+**The unique traits of digital goods listed above mean they’re particularly susceptible to market failures**. People can’t produce digital goods and sell them in the same way as other goods, and so Adam Smith’s parable of the baker breaks down. Even in cases of private benefit, these can be subtle and not apparent to key decision-makers (e.g. efficiency gains in ways that aren't apparent upfront), meaning there’s still a disinclination to invest. They might also take the form of industry wide benefits (e.g. from innovation potential), which are hindered by the dynamics of collective action problems[^4].
+
+
+<!-- Footnotes themselves at the bottom. -->
```
Comment (rufuspollock) [inline content/notes/digital-goods-are-different.md:24]: This html comment will break the build ...

File: content/notes/digital-goods-are-different.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+     This means that one person using the good doesn’t reduce it for others. If I eat the bread you can’t also eat it, but if I use a piece of data you can use it also.
+
+[^2]:
+     The cost of producing one extra unit of a good.
+
+[^3]:
+     Opportunity costs are the other options for spending your time and resources passed up.
+
+[^4]:
+    [ https://www.e-education.psu.edu/geog30/node/342](https://www.e-education.psu.edu/geog30/node/342)
```
Comment (rufuspollock) [inline content/notes/digital-goods-are-different.md:37]: Don't think you need the double link. The raw link itself will auto-convert.

**Comment by rufuspollock** [comment]
@theo-cox the build failed so this won't have deployed i don't think. See my comments for why.

**Review by rufuspollock — APPROVED** [review]
As usual, very decent first pass - and done rapidly.

My one question here is we have written this before - and i suspect a bit better (e.g. i wrote this up in open revolution or in several of my talks).

This version is a a big jargon-y / economics-y. Eg. we tend to use term "costless copying" vs "marginal cost is zero".

Of course, happy to go ahead but wanted to flag that for such a core piece think we maybe could be a bit simpler and clearer.

**Comment by theo-cox** [comment]
> As usual, very decent first pass - and done rapidly.
> 
> My one question here is we have written this before - and i suspect a bit better (e.g. i wrote this up in open revolution or in several of my talks).
> 
> This version is a a big jargon-y / economics-y. Eg. we tend to use term "costless copying" vs "marginal cost is zero".
> 
> Of course, happy to go ahead but wanted to flag that for such a core piece think we maybe could be a bit simpler and clearer.

@rufuspollock the other stuff was incredibly lengthy and narrative from what I saw. Happy to reuse but it didn't strike me as particularly accessible/useful either for our purposes, but almost from the other end of the spectrum

**Pull request merged**